tests/migrator/test_books.py

```diff
@@ -1042,13 +1042,13 @@ def test_extensions(app):
             <datafield tag="925" ind1=" " ind2=" ">
                 <subfield code="i">applicable at CERN</subfield>
                 <subfield code="p">Expert ICS-25.160</subfield>
-                <subfield code="z">Reviewed in December 2019</subfield>
+                <subfield code="z">20040621</subfield>
```

Comment:
are there any examples of records from legacy with this date format ?
the tests were written to mirror the real records, f.e. https://cds.cern.ch/record/2690790
I am asking because in this case we have to somehow normalise the data

Reply:
Your are right, I checked the records and all of them have `Reviewed in XXX`. I will normalise it, thanks!